src/tests/system/tests/test_passkey.py

```diff
@@ -3,11 +3,67 @@
 
 :requirement: passkey
 
+Passkeys allow users to authenticate without having to enter a username or password,
+or provide any additional authentication factor.
+This technology aims to replace legacy authentication mechanisms such as passwords.
+
+The objective is to use a passkey to locally authenticate a user against centralized identity management system.
+For that purpose an integration with a server like AD/Samba or IPA or LDAP is needed.
+
 The passkey solution only enables to authenticate in a system where the
-FIDO2 key is connected physically.This could be su login, the GUI,
+FIDO2 key is connected physically.This could be su login, the GDM login,
 or even ssh<user>@localhost.
+
+We can't support remote authentication (ssh) because there isn't any way of doing the remote authentication
+when the key is attached to your laptop.
+
 Here, passkey support is tested with su, tests are running with
 umockdev, not with a physical key.
```

Comment:
Get rid of line break?

Reply:
Clarification, both lines are 99 characters line, it just looks funny breaking this line like that when 17 isn't broken.